backend/firestore.rules

```diff
@@ -19,7 +19,8 @@ service cloud.firestore {
       allow read, write: if userCanAccess(resource);
 
       match /{documents=**} {
-        allow read, write: if userCanAccess(getAfter(/databases/$(database)/documents/scenarios/$(scenario)));
+        allow read, delete, update: if userCanAccess(get(/databases/$(database)/documents/scenarios/$(scenario)));
+        allow create: if userCanAccess(getAfter(/databases/$(database)/documents/scenarios/$(scenario)));
```

Comment:
This makes more sense.  For documents that already exist, reading, deleting, and updating can occur within a batch in a `get`.  For documents that don't exist yet, create can occur within a `getAfter`.  Tested all existing use cases with reads, writes, updates, deletes, etc in the multi-facility page and this all worked like a charm 👌